### PR TITLE
Polished docs: fixed grammar and wording issues

### PR DESCRIPTION
**Pull request by sadler-elena**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by orpiske — APPROVED** [review]

File: docs/user-manual/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
 
-You can get started with Apache Camel using different ways, such as:
+You can get started with Apache Camel in a variety of ways, such as:
 
 - Using online Project generators
 - Using the Camel CLI (command line)
 - Using Camel Karavan (visual UI Camel designer)
 
-And these approaches requires more knowledge to perform:
+And more advanced methods:
```
Comment (davsclaus) [inline docs/user-manual/modules/ROOT/pages/getting-started.adoc:9]: I dont think this is advanced

Comment (sadler-elena) [reply docs/user-manual/modules/ROOT/pages/getting-started.adoc:9]:
```suggestion
And some more alternative methods:
```

Comment (davsclaus) [reply docs/user-manual/modules/ROOT/pages/getting-started.adoc:9]: LGTM

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
Thanks for helping with docs

**Pull request merged**